Failed to build opentelemetry-cpp with OTLP HTTP exporter

**Describe your environment**
OS: Debian 10 
Release version: opentelemetry-cpp v1.1.0
Compiler: g++ 8.3.0

**Steps to reproduce**

> cmake ../ -DWITH_OTLP=ON -DWITH_OTLP_HTTP=ON -DBUILD_TESTING=OFF
>make -j

**What is the expected behavior?**
Builds successfully

**What is the actual behavior?**
The build failed with the following error
>opentelemetry-cpp/exporters/otlp/src/otlp_http_client.cc:292:40: error: cannot convert 'const string' {aka 'const std::__cxx11::basic_string<char>'} to 'const unsigned char*'
         google::protobuf::Base64Escape(bytes, &base64_value);

>opentelemetry-cpp/exporters/otlp/src/otlp_http_client.cc:31:58: note:   initializing argument 1 of 'int google::protobuf::Base64Escape(const unsigned char*, int, char*, int)'
 LIBPROTOBUF_EXPORT int Base64Escape(const unsigned char *src, int slen, char *dest, int szdest);

**Aditional Context**
It seems that the function Base64Escape was being called with the incorrect set of arguments. It was called with bytes, which is an `std::string` instead of a `const unsigned char *`.

--- a/exporters/otlp/src/otlp_http_client.cc
+++ b/exporters/otlp/src/otlp_http_client.cc
@@ -21,6 +21,7 @@
 #include "google/protobuf/stubs/common.h"
 
 #if defined(GOOGLE_PROTOBUF_VERSION) && GOOGLE_PROTOBUF_VERSION >= 3007000
+#  include "google/protobuf/stubs/stringpiece.h"
 #  include "google/protobuf/stubs/strutil.h"
 #else
 #  include "google/protobuf/stubs/port.h"
@@ -28,7 +29,7 @@ namespace google
 {
 namespace protobuf
 {
-LIBPROTOBUF_EXPORT int Base64Escape(const unsigned char *src, int slen, char *dest, int szdest);
+LIBPROTOBUF_EXPORT int Base64Escape(StringPiece src, std::string *dest);
 }  // namespace protobuf
 }  // namespace google
 #endif

--- a/exporters/otlp/src/otlp_http_client.cc
+++ b/exporters/otlp/src/otlp_http_client.cc
@@ -29,7 +29,7 @@ namespace google
 {
 namespace protobuf
 {
-LIBPROTOBUF_EXPORT int Base64Escape(StringPiece src, std::string *dest);
+LIBPROTOBUF_EXPORT void Base64Escape(StringPiece src, std::string *dest);
 }  // namespace protobuf
 }  // namespace google
 #endif

--- a/exporters/otlp/src/otlp_http_client.cc
+++ b/exporters/otlp/src/otlp_http_client.cc
@@ -21,7 +21,6 @@
 #include "google/protobuf/stubs/common.h"
 
 #if defined(GOOGLE_PROTOBUF_VERSION) && GOOGLE_PROTOBUF_VERSION >= 3007000
-#  include "google/protobuf/stubs/stringpiece.h"
 #  include "google/protobuf/stubs/strutil.h"
 #else
 #  include "google/protobuf/stubs/port.h"

